index.html

```diff
@@ -55,6 +54,228 @@ <h2 class="green">I’m a Full Stack developer from Microverse</h2>
+        </div>
+        
+      </div>
+
+      <!-- Project 3 -->
+
+      <div class="project">
+        <div class="project-image">
+          <img src="" alt="" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Contact form
+          </h3>
+        </div>
+        <ul class="project-built-with">
+          <li><a href="">Ruby on Rails</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">Javascript</a></li>
+          <li><a href="">HTML</a></li>
+        </ul>
+
+        <div class="project-buttons">
+          <a href="">See project</a>
+        </div>
+        
+      </div>
+
+      <!-- Project 4 -->
+
+      <div class="project">
+        <div class="project-image">
+          <img src="" alt="" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Desktop version
+          </h3>
+        </div>
+        <ul class="project-built-with">
+          <li><a href="">Ruby on Rails</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">Javascript</a></li>
+          <li><a href="">HTML</a></li>
+        </ul>
+
+        <div class="project-buttons">
+          <a href="">See project</a>
+        </div>
+        
+      </div>
+
+      <!-- Project 5 -->
+
+      <div class="project">
+        <div class="project-image">
+          <img src="" alt="" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Application deploy
+          </h3>
+        </div>
+        <ul class="project-built-with">
+          <li><a href="">Ruby on Rails</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">Javascript</a></li>
+          <li><a href="">HTML</a></li>
+        </ul>
+
+        <div class="project-buttons">
+          <a href="">See project</a>
+        </div>
+        
+      </div>
+
+    </div>
+  </section>
+
+  <section class="about-myself">
+
+    <div class="about-myself-text">
+      <header>
+        <h2 class="green">
+          About me
+        </h2>
+      </header>
+
+      <p>
+          I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hesitate to contact me.
+      </p>
+    </div>
+
+    <div class="project-buttons see-resume-button">
+      <a href="">Get my resume</a>
+    </div>
+    
+    <div class="competences">
+      
+      <div class="languages">
+        <div class="competences-image">
+          <img src="images/Icons/languages.png" alt="languages" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Languages
+          </h3>
+        </div>
+        <ul class="list-competences">
+          <li><a href="">Javascript</a></li>
+          <li><a href="">Ruby</a></li>
+          <li><a href="">HTML</a></li>
+          <li><a href="">CSS</a></li>
+        </ul>
```

Comment:
- Just as the last reviewer mentioned, please ensure that your skills and technology cards are styled precisely as in the Figma template. Please fix this 🙏 

Your project:
- Please add a border-radius to the list of technologies 🙏 
<img width="296" alt="qergqerwg" src="https://user-images.githubusercontent.com/73607512/227745334-cb706873-dbfe-4a00-92dc-00e58203d177.PNG">

Design template:
![zsas](https://user-images.githubusercontent.com/73607512/227745375-f89a173d-84cc-498a-842c-5a833e775029.png)

Reply:
Done @mckent05 @mahbubul14